build.gradle.kts

```diff
@@ -165,7 +166,7 @@ dependencies {
     implementation("org.owasp:dependency-check-gradle:latest.release")
     implementation("com.netflix.nebula:gradle-contacts-plugin:latest.release")
     implementation("com.netflix.nebula:gradle-info-plugin:latest.release")
-    implementation("com.netflix.nebula:nebula-release-plugin:latest.release")
+    implementation("com.netflix.nebula:nebula-release-plugin:18.0.+") // "Git user.name is not set."
```

Comment:
```suggestion
    implementation("com.netflix.nebula:nebula-release-plugin:$nebulaVersion") {
        because("Git user.name is not set.");
    }
```